timereport/chalicelib/lib/factory.py

```diff
@@ -1,5 +1,5 @@
 from datetime import datetime, timedelta
-from timereport.model.event import create_event
+from ..model.event import create_event
```

Comment:
I prefer the more direct import `timereport.model.event` over this relative import

Reply:
good point - must have been my IDE acting up and telling me it can't import create_event because of that path. Fixed it now